packages/utils/src/__tests__/period/period/period.test.js

```diff
@@ -0,0 +1,246 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import MockDate from 'mockdate';
+import moment from 'moment';
+
+import {
+  dateStringToPeriod,
+  findCoarsestPeriodType,
+  getCurrentPeriod,
+  isCoarserPeriod,
+  isFuturePeriod,
+  isValidPeriod,
+  momentToPeriod,
+  parsePeriodType,
+  PERIOD_TYPES,
+  periodToTimestamp,
+  periodToType,
+} from '../../../period/period';
+import { testConvertToPeriod } from './testConvertToPeriod';
+import { testGetPeriodsInRange } from './testGetPeriodsInRange';
+import { testPeriodToDisplayString } from './testPeriodToDisplayString';
+import { testComparePeriods } from './testComparePeriods';
```

Comment:
This commit seems large but don't worry - I just extracted the longer test suites from `period.test.js` to the above files! It' s a pattern we commonly use to have more manageable and readable test files

The convention we use for those file names is:
`test{{TestedUnit}}`/`test{{TestedFunctionality}}`